os/include/tinyara/sched.h

```diff
@@ -86,11 +86,14 @@
 /********************************************************************************
  * Pre-processor Definitions
  ********************************************************************************/
-#ifdef CONFIG_ARMV7M_MPU
-#define REG_RNR		0
-#define REG_RBAR	1
-#define REG_RASR	2
-#define MPU_REGS	3
+#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(CONFIG_ARMV7M_MPU)
+enum {
+	REG_RNR,
+	REG_RBAR,
+	REG_RASR
+};
```

Comment:
This is mpu specific definitions. Let's move them into mpu.h.

Reply:
I will make this change in my next PR for mpu changes.